base/stl_helpers.hpp

```diff
@@ -51,6 +54,16 @@ void SortUnique(vector<T> & v)
   v.erase(unique(v.begin(), v.end()), v.end());
 }
 
+// Sorts according to |comp| and removes duplicate entries according to |pred| from |v|.
+// Note. If several entries are equal according to |pred| an arbitrary entry of them
+// is left in |v| after a call of this function.
+template <typename T, typename TComp, typename TPred>
+void SortUnique(vector<T> & v, TComp && comp, TPred && pred)
```

Comment:
s/comp/less/g
s/pred/equals/g
s/TComp/TLess/g
s/TPred/TEquals/g

Because of both functors are predicates.